Frontend/CO.CDP.OrganisationApp/Pages/Registration/BuyerSelectDevolvedRegulation.cshtml.cs

```diff
@@ -10,7 +11,7 @@ public class BuyerSelectDevolvedRegulationModel(ISession session) : Registration
     public override string CurrentPage => BuyerSelectDevolvedRegulationPage;
 
     [BindProperty]
-    [NotEmpty(ErrorMessage = "Select a devolved region")]
+    [NotEmpty(ErrorMessage = nameof(StaticTextResource.OrganisationRegistration_BuyerSelectDevolvedRegulation_ValidationErrorMessage))]
```

Comment:
I think this won't work - for me the `NotEmpty` validator is not accepting this style of doing things which would mean this validation error message literally shows on screen as `OrganisationRegistration_BuyerSelectDevolvedRegulation_ValidationErrorMessage`

Reply:
@dpatel017 I have sorted this out on my branch in 3087763c94c35c2367c521a947a065e0d0119753

If you `git cherry-pick 3087763c94c35c2367c521a947a065e0d0119753`  to bring it across to yours, and then you will need to do the following as well:

```suggestion
    [NotEmpty(ErrorMessageResourceName = nameof(StaticTextResource.OrganisationRegistration_BuyerSelectDevolvedRegulation_ValidationErrorMessage), ErrorMessageResourceType = typeof(StaticTextResource))]
```